src/views/List.jsx

```diff
@@ -13,29 +16,50 @@ export function List({ data }) {
 		setSearchTerm('');
 	};
 
+	const handleClick = (selectedPath) => {
+		navigate(selectedPath);
+	};
+
 	const filteredData = data.filter((item) => {
 		const itemName = item.name.toLowerCase();
 		return itemName.includes(searchTerm.toLowerCase());
 	});
 
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
-			<form>
-				<label htmlFor="itemFilter">
-					Search for an item:
-					<input
-						type="text"
-						id="itemFilter"
-						name="itemFilter"
-						value={searchTerm}
-						onChange={handleChange}
-					/>
-				</label>
-				{searchTerm && <button onClick={reset}>Reset</button>}
-			</form>
+			{!listPath && (
+				<>
+					<h2>
+						You haven't selected a list yet. Click below to select a list.
+					</h2>
+					<button onClick={() => handleClick('/')}>Select a list</button>
+				</>
+			)}
+			{listPath && data.length === 0 && (
```

Comment:
Question about best practices for this sort of expression:

would ln 38 be better written as `{**(**listPath && data.length === 0**)** && (` ?

It seems to work just fine in the format it is in, but it sort of seems like it would be 'safer' to use parenthesis to make sure it is always evaluated in the order of operations that suit our needs.

Reply:
Hi @sdmisra, the parenthesis are not needed for this expression, but it's usually best to render conditional UI using ternaries instead of conditional chaining like this. So instead of `{listPath && data.length === 0 && <Component />}`, using `{listPath && data.length === 0 ? <Component /> : null}` instead. This [ESLint rule](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-leaked-render.md) explains why it's best to use the second approach.

Reply:
@luisaugusto That makes sense - I can make that change to List.jsx and add it to this PR. Thank you for the clarification!